code/datums/wounds/_wounds.dm

```diff
@@ -303,7 +309,8 @@
   * * mob/user: The user examining the wound's owner, if that matters
   */
 /datum/wound/proc/get_examine_description(mob/user)
-	return "<B>[victim.p_their(TRUE)] [limb.name] [examine_desc]!</B>"
+	. = "[victim.p_their(TRUE)] [limb.name] [examine_desc]"
+	. = severity <= WOUND_SEVERITY_MODERATE ? "[.]." : "<B>[.]!</B>"
```

Comment:
+=?

Reply:
it wraps around the above line, look again. bobbeh told me to do this because of micro-optimizations, i think it's silly but i guess it's more compact